package.json

```
@@ -36,7 +36,8 @@
     "unit": "jest --coverage",
     "test": "yarn lint && yarn unit",
     "test:integration": "yarn lint && jest --config=jest-integration.config.js",
-    "watch": "jest --watch",
+    "watch": "NODE_ENV=development npx nodemon .",
```

Two things. `npx` is unnecessary since it is a NPM command and we are using yarn. And the environment variable assignment only works on unix-like systems. So windows systems won't run it.

You can use `cross-env` to get around the issue with environment variables:
```suggestion
    "watch": "cross-env NODE_ENV=development nodemon .",
```